rosidl_typesupport_cpp/src/type_support_dispatch.hpp

```diff
@@ -108,6 +108,7 @@ get_typesupport_handle_function(
       return ts;
     }
   }
+  rcutils_reset_error();
```

Comment:
Why reset here? It doesn't seem like we should.

Reply:
I remember putting it there to squash a warning but I don't recall what for. I've removed it in 05b9f45c3889b8a87831a3cd758bc9d1303d0106